controllers/session_test.go

```diff
@@ -116,7 +116,7 @@ var _ = Describe("SessionController", func() {
 			})
 
 			Context("given an INVALID params", func() {
-				Context("when the user does NOT exist", func() {
+				Context("given NO user exist", func() {
```

Comment:
Small synatx issue 🙈 :

```suggestion
				Context("given NO user exists", func() {
```